spec/graphql-pundit/authorization_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.shared_examples 'auth field' do |do_raise|
+  before do
+    [CarPolicy, ChineseCarPolicy].each do |policy|
+      allow_any_instance_of(policy).to receive(:name?).and_return(auth_result)
+      allow_any_instance_of(policy).to(
+        receive(:display_name?).and_return(auth_result)
+      )
+    end
+  end
+
+  context 'authorized' do
+    let(:auth_result) { true }
```

Comment:
I would name this `expected_auth_result`, but it's nothing that has to be changed for the approval